db/migrate/20240314020558_create_collections.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCollections < ActiveRecord::Migration[7.1]
+  def change
+    create_table :collections do |t|
+      t.string :name
+      t.integer :user_id
+      t.integer :city_id
```

Comment:
Since you have `presence: true` in the validation, I suggest making these non-nullable columns and adding the foreign keys for better data integrity.